File: 프로그래머스/문자열 내 P와 Y의 개수.py
def solution(s):
    count = 0
    
    for c in s:
        if c == "p" or c == "P":
            count += 1
        elif c == "y" or c == "Y":
            count -= 1
    
    if count == 0:
        return True
    else:
        return False

File: 프로그래머스/문자열 내 P와 Y의 개수.java
class Solution {
    boolean solution(String s) {
        char[] input = s.toCharArray();
        int count = 0;
        for (char ch : input) {
            if (ch == 'p' || ch == 'P')
                count++;
            else if (ch == 'y' || ch == 'Y')
                count--;
        }
        return (count == 0) ? true : false;
    }
}